omod/src/main/webapp/resources/scripts/visit/constants.js

```diff
@@ -1,7 +1,12 @@
 angular.module('constants', [])
     .value('DatetimeFormats', {
         date: "d-MMM-yy",
+        dateLocalized: { year: 'numeric', month: 'short', day: 'numeric' },
         time: "hh:mm a",
+        // TODO: fix timeLocalized:
+        //   'hour12' override is necessary because we're using `fr` in Haiti instead of `fr_HT`.
+        //   This override will prevent time from displaying correctly in places that use a 24 hour clock.
```

Comment:
Confused by this comment... it sounds like this override is meant to prevent time from being display correctly, which I assume is not the case? @brandones  @mseaton ?

Reply:
We always display 12-hour time (AM/PM). This is okay in Haiti, Mexico, Peru, Sierra Leone, and Liberia; but it is incorrect in Rwanda, Lesotho, and Kazakhstan, which all use the 24-hour clock. Fortunately the PIH EMR isn't used in those latter countries.

Normally if you use the JavaScript date/time localization functions they will render in 12 or 24 hour format depending on the locale. However, `fr` uses the 24-hour clock (as in France), but we use `fr` in Haiti instead of `fr_HT`. By setting `hour12: true`, we force it to show 12-hour time, even for the `fr` locale.